docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[dataframe-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-wizard-limitations]]
+=== {dataframe-cap} wizard limitation
+
+If you have fields that contain arrays, you cannot select these fields in the source 
+index table in the {kib} {dataframe} wizard. You can, however, use those fields 
+when you group and aggregate your data in the wizard.
+
+[float]
+[[df-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+
+[float]
+[[df-kibana-limitations]]
+=== Displaying {dataframe-transforms} in {kib} limitation
```

Comment:
Change title to `{dataframe-transforms} in {kib} will only display up to 100 {dataframe-transforms}`

Reply:
(note - this will increase to 1000 with 7.2.1)